### DISCUSSION TITLE
Created a Reinstatement Request and I am scared I won't get a fast response from GitHub

### DISCUSSION THREAD

**Discussion by AlexandruPaulescu** [community/community#84944] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I need my account back from being marked as spam because my work kinda depends on this. I can not do any Git related actions. And I saw that most people get responses in days unfortunately so I am scared I won't be getting it as soon as I need it. Anyone know any action I can take so they respond quickly? Thanks.

**Comment by ibra-kdbra** [comment]
Unfortunately, it may take some time for GitHub Support to respond due to the volume of requests they receive. Be patient and wait for their response.
I had my other account suspended for like one and half month, fortunately I had this account which they reset but it's okay, using both now, It's sometimes frustrating to happen, words can't describe this, GOOD LUCK for getting it faster.

**Reply by AlexandruPaulescu** [reply]
I'm hoping for a GitHub's stuff to help me on this one.. Thanks for sharing your experience haha

**Reply by AppiumGuide** [reply]
please don't keep any hopes from github support team, issue will not be resolve , i wait more than months.

i delete my repos & cannot delete the account, stop using the account that's it, only solve.

**Comment by CodeWizard0214** [comment]
Hi, @AlexandruPaulescu 
I faced same issue, Did you solved this issue?
to recovery my account, What should I do?
I made ticket with this issue but there is no reply yet. How many day did you wait?
Thank you

**Comment by Kareem-dev** [comment]
Hey man, I'm in the same situation, although my account was completely suspended for no reason so I can't even access it. I was wondering if you've received any response and how long it took for your reinstatement request to get answered, since I'm currently on my 25th or 26th day (I lost count... since the 5th of June) of not being able to work at all.

**Reply by AppiumGuide** [reply]
please don't keep any hopes from github support team, issue will not be resolve , i wait more than months.

i delete my repos & cannot delete the account, stop using the account that's it, only solve.

**Comment by baharbaba** [comment]
I've been waiting for a response for 2 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.

**Reply by ibra-kdbra** [reply]
Sometimes you need that..

**Reply by saahilkhanna** [reply]
did it end up working for you. Been two months and I have not got any indication of response

**Comment by AppiumGuide** [comment]
please don't keep any hopes from github support team, issue will not be resolve , i wait more than months.

i delete my repos & cannot delete the account, stop using the account that's it, only solve.
